lineage/StateDistribution.py

```diff
@@ -33,7 +33,7 @@ def pdf(self, tuple_of_obs):  # user has to define how to calculate the likeliho
         # distribution observations), so the likelihood of observing the multivariate observation is just the product of
         # the individual observation likelihoods.
 
-        bern_ll = sp.bernoulli.pmf(k=tuple_of_obs[0], p=self.bern_p)  # bernoulli likelihood
+        bern_ll = self.bern_p**(tuple_of_obs[0]) * (1.0 - self.bern_p)**(tuple_of_obs[0])
```

Comment:
@shak360    I think this is what's wrong. `tuple_of_obs[0]` is an array, so you're turning `bern_ll` into an array.